Services/Masterdata/Services.Masterdata/Services/MasterdataService.cs

```diff
@@ -56,6 +62,12 @@ public Task<MasterdataModel> AddMasterdata(string masterdataType, MasterdataCrea
         return _masterdataRepository.AddMasterdata(masterdataType, model);
     }
 
+    public Task<ICollection<MasterdataKeyModel>> AddMasterdataKeys(string masterdataType, string masterdata, ICollection<MasterdataKeyCreateModel> model)
+    {
+        //model.Sanitize(htmlSanitizer);
```

Comment:
Why is the sanitization commented out? Is this checked in another location? If so, remove this code, if not, the keys _must_ be sanitized to prevent any malicious user input